### ISSUE TITLE
pt and drt example

### ISSUE THREAD

**Issue by ywf** [matsim-org/matsim-code-examples#790]
hello，can each plan support multi legs ? for example.the trip is separated two sections,the first is pt and the second is drt, is it OK?
if the plan file is defined as follows:
`
        <person id="passenger_0" employed="no">
		<plan selected="yes">
			<act type="dummy" link="114" end_time="00:00:00"/>
			**<leg mode="drt"/>
			<leg mode="pt"/>**
			<act type="dummy" link="349"/>
		</plan>
	</person>
`

**Comment by balacmi** [comment]
Yes, the plan can support multiple legs within a trip. In your example, if you use the Intermodal SwissRailRaptor for public transport trips, you will also get a _pt interaction_ activity that is used to maintain alternating leg/activity representation of the plan. This interaction activity is normally not scored. 

I hope this helps.

**Comment by ywf** [comment]
thanks for your reply，SwissRailRaptor  can support the transfer between many pt lines , can SwissRailRaptor  support the transfer between pt and multiDrt? thanks
